rclpy/rclpy/publisher.py

```diff
@@ -12,10 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import List
 from typing import TypeVar
 
+from rclpy.callback_groups import CallbackGroup
 from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
 from rclpy.qos import QoSProfile
+from rclpy.qos_event import PublisherEventCallbacks
+from rclpy.qos_event import QoSEventHandler
```

Comment:
I think we both got the naming convention wrong. In `rclcpp` it's `QOSEventHandler`, here it's `QoSEventHandler`, but the Google style guide would've wanted `QosEventHandler`. Let's change it?

Reply:
The existing python codebase uses `QoS` (`AaA`) for the functionality that was already here.  I'm going to keep it this way to match the existing convention.

See https://github.com/ros2/rclpy/blob/master/rclpy/rclpy/qos.py#L21